CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2019 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+cmake_minimum_required(VERSION 3.5)
+project(whole-body-estimators LANGUAGES CXX
+                              VERSION 0.1)
+                             
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+option(BUILD_SHARED_LIBS "Build libraries as shared as opposed to static" ON)
+
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+
+set(YARP_REQUIRED_VERSION 3.0.1)
+
+find_package(YARP REQUIRED)
+if(${YARP_VERSION} VERSION_LESS ${YARP_REQUIRED_VERSION})
+    message(FATAL_ERROR "YARP version ${YARP_VERSION} not sufficient, at least version ${YARP_REQUIRED_VERSION} is required.")
+endif()
```

Comment:
Just to understand, why don't you simply write:
~~~
find_package(YARP 3.0.1 REQUIRED)
~~~
?

Reply:
True. Mindless ctrl+c ctrl+v.

Reply:
Addressed in 82d978f1892d49beb5bdc29df0f343a153b2678a